_layouts/default.html

```diff
@@ -4,7 +4,54 @@
   {% include head.html %}
 
   <body>
+    <svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I don't understand why it's better to have inline svg which is rendered on each pages instead of external ones referenced through `<img/>` tags, which are downloaded once and cached, and which also benefit from site wide `img` css rules.
(This is a real interrogation, I should be missing a point here because this is not the first time I see this)

Reply:
The main advantage of inline svg is that it can be manipulated with css. I was intending to add hover effect as they are used as link icons and that would prevent from having two different files for each and I don't think they cost really much.

The actual best solution might be to have an external svg sprites, could be one of our next step.

Reply:
:+1: